### PR TITLE
[TK-05573] switch to yaml for conductor config

### PR DESCRIPTION
**Pull request by neonphog**

### CODE REVIEW

File: crates/holochain/src/conductor/config.rs [hunk 1]
```diff
@@ -84,24 +84,25 @@ pub struct ConductorConfig {
...
-/// helper fnction function to load a `Config` from a toml string.
-fn config_from_toml<'a, T>(toml: &'a str) -> ConductorResult<T>
+/// helper fnction function to load a `Config` from a yaml string.
+fn config_from_yaml<T>(yaml: &str) -> ConductorResult<T>
 where
-    T: Deserialize<'a>,
+    T: DeserializeOwned,
 {
-    toml::from_str::<T>(toml).map_err(ConductorError::DeserializationError)
+    //yaml::from_str::<T>(yaml).map_err(ConductorError::DeserializationError)
```
Comment (neonphog) [inline crates/holochain/src/conductor/config.rs:92]:
```suggestion
```

File: crates/holochain/src/conductor/config.rs [hunk 2]
```diff
@@ -157,32 +157,39 @@ pub mod tests {
...
+    dpki:
+      instance_id: some_id
+      init_params: some_params
+
+    admin_interfaces:
+      -
+        driver:
+          type: websocket
+          port: 1234
+
```
Comment (freesig) [inline crates/holochain/src/conductor/config.rs:179]: Is this a array of length one where the value is a dictionary of two key pairs?
Shouldn't it be:
```suggestion
      - driver:
          type: websocket
          port: 1234

```
Or does it make no difference?

**Review by freesig — APPROVED** [review]
:+1: :rocket:

File: crates/holochain/src/conductor/config.rs (hunk 2 above)
Comment (neonphog) [reply crates/holochain/src/conductor/config.rs:179]: Yeah, I think either way is fine, but using one less line may be more canonical.

File: crates/holochain/src/conductor/config.rs [hunk 3]
```diff
@@ -157,32 +157,38 @@ pub mod tests {
...
+    admin_interfaces:
+      - driver:
+          type: websocket
+          port: 1234
+
+    network:
+      transport_pool:
+        -
+          type: proxy
```
Comment (neonphog) [inline crates/holochain/src/conductor/config.rs:182]:
```suggestion
        - type: proxy
```

**Pull request merged**